spanner/api/Spanner.Samples.Tests/CreateBackupTest.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google Inc.
+// Copyright 2021 Google Inc.
```

Comment:
nit: Please revert this change, we don't update the copyright year when we make modifications, the code is copyrighted since it was first published.

Reply:
Fixed.